src/lib/containers/ForumSearch.tsx

```diff
@@ -0,0 +1,147 @@
+                <Typography variant='body2'>Search the full text of posts, replies, and titles</Typography>
+            </div>
+        )
+    }
+
+    const onLoadMore = async () => {
+        const searchResponse = await forumSearch(
+            {
+                searchString: searchInput,
+                nextPageToken: searchResult?.nextPageToken
+            },
+            props.forumId,
+            accessToken)
+        setSearchResult(searchResponse)
+        if (matchList) {
+            setMatchList([...matchList, ...searchResponse.matches])
+        }
+    }
+
+    return (
+        <div className="bootstrap-4-backport ForumSearch">
+            <div>
+                <FontAwesomeIcon
+                    size={'sm'}
+                    icon={faSearch}
+                    className="SearchIcon"
+                />
+                <input
+                    role='textbox'
+                    type='search'
+                    className={`SearchBar  ${searchResult ? 'SearchBarResult' : ''}`}
+                    placeholder='Search discussions'
+                    value={searchInput}
+                    onChange={event => {
+                        setSearchInput(event.target.value)
+                    }}
+                    onKeyDown={(event: any) => {
+                        if (event.key === 'Enter') {
+                            onSearch()
+                        }
+                    }}
+                />
+                {searchInput && (
+                    <FontAwesomeIcon
+                        size={'sm'}
+                        icon={faTimes}
+                        role="button"
+                        title="Clear Search"
+                        className="ClearSearchIcon"
+                        onClick={() => {
+                            setSearchInput('')
+                        }}
+                    />
+                )}
+            </div>
+            {
+                noSearchResult && <NoSearchResultComponent />
+            }
+            {
+                isLoading ?
+                    <div style={{ textAlign: 'center' }}>
+                        <SynapseSpinner size={40} />
+                    </div> :
+                    matchList &&
+                    <>
+                        {props.projectId && (
+                            <Typography variant='body2'>Results for '{searchInput}' in {entity?.name}</Typography>
+                        )}
+                        {matchList.map(entity =>
+                            <div key={`${entity.forumId}-${entity.threadId}-${entity.replyId}`}>
+                                <DiscussionReply threadId={entity.threadId} replyId={entity.replyId} />
+                            </div>
+                        )}
+                    </>
+            }
+            {
+                searchResult?.nextPageToken &&
+                <div className='text-center'>
```

Comment:
Not sure if centering is correct, but I think Button variant should be "primary"

Reply:
Should the button not be centered?

Reply:
I would double check other instances of paged results, but I don't think this is typically centered (but I might be wrong)